src/test/java/com/mercadopago/client/order/OrderClientTest.java

```diff
@@ -163,4 +164,20 @@ void deleteSuccess() throws MPException, MPApiException, IOException {
         Assertions.assertNotNull(order);
         Assertions.assertEquals(orderId, order.getId());
     }
+
+    @Test
+    void captureSuccess() throws MPException, MPApiException, IOException {
+
+        //Mock HttpClient
+        HttpResponse response = MockHelper.generateHttpResponseFromFile(CAPTURE_ORDER_RESPONSE_FILE, HttpStatus.OK);
+        Mockito.doReturn(response).when(HTTP_CLIENT).execute(any(HttpRequestBase.class), any(HttpContext.class));
+
+        //when
+        Order order = client.capture("123");
+
+        //then
+        Assertions.assertNotNull(order);
+        Assertions.assertEquals(order.getStatus(), "processed");
+
```

Comment:
Remover espaços no final, gera code smells
```suggestion
        Assertions.assertEquals(order.getStatus(), "processed");
```